packages/regl-worldview/src/utils/Dimensions.js

```diff
@@ -26,7 +28,8 @@ class ResizeObserverMock {
   }
   unobserve() {}
 }
-const ResizeObserverImpl = process.env.NODE_ENV === "test" ? (ResizeObserverMock: any) : ResizeObserver;
+const ResizeObserverImpl =
+  process.env.NODE_ENV === "test" || inWebWorker() ? (ResizeObserverMock: any) : ResizeObserver;
```

Comment:
I'm not sure about this. Does it make sense to have a `Dimensions` object/component inside the worker? I suspect we don't need support here. (If we don't, we probably don't need the implementation of `inWebWorker`.)

Reply:
We don't need `Dimensions` inside workers. But since I haven't make every feature of `Worldview` to be worker-friendly yet, not having this check ends up causing a runtime error because `ResizeObserver` is not defined. I'm going to get back to this in a future PR

Reply:
Fair enough. Maybe add a TODO to remove the worker check if we think we'll fix the import order later, but not a big deal.